### PR TITLE
Added choice and destination node to fork-jam

### PR DESCRIPTION
**Pull request by spaetsch**

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/fork-jam/not_dreaming.md [hunk 1]
```diff
@@ -8,4 +8,6 @@ Do you:
 
 [Drink from it?](./choco-chalice/choco-chalice.md)
 
+[Smash the altar](../altar-smash/altar-smash.md)
```
Comment (casey-collab) [inline english/fork-jam/not_dreaming.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/fork-jam/altar-smash/altar-smash.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+You smash the graham cracker altar, and the chalice and biscuit debris vanish in a puff of powdered sugar.
```
Comment (casey-collab) [inline english/fork-jam/altar-smash/altar-smash.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

**Comment by casey-collab** [comment]
Looks great! :melon:

**Pull request merged**